lang/funcs/funcgen/pkg.go

```diff
@@ -18,13 +18,25 @@
 package main
 
 import (
+	"bytes"
+	"errors"
+	"fmt"
 	"io/ioutil"
 	"log"
+	"os/exec"
 	"path/filepath"
+	"regexp"
+	"strings"
 
+	"github.com/iancoleman/strcase"
 	yaml "gopkg.in/yaml.v2"
 )
 
+var (
+	validSignature = regexp.MustCompile(`^func (?P<name>[A-Z][a-zA-Z0-9]+)\((?P<args>([a-zA-Z]+( (string|bool|float64)|, )){0,})\) (?P<return>(string|float64|bool)|\((string|float64|bool), error\))$`)
+	excludedErr    = errors.New("Function is excluded")
```

Comment:
`errExcluded` instead please, and error messages are not capitalized (test failure)